GFSv16.1.5 - GSI update to include IASI Metop-C in operations

KateFriedman-NOAA opened the pull request:
RFC 8823 - On WCOSS, update GFS to v16.1.5 which includes:
* Turn on the active assimilation of Metop-C IASI (Infrared Atmospheric Sounding Interferometer) data with correlated observation errors
* Update correlated observation errors for Metop-B IASI, N20 CrIS (Cross-track Infrared Sounder), and NPP CrIS
* Put in a minimum threshold for saturation specific humidity to avoid potential minimization issues.
* Add the 2022 CO2 projection files.

Implemented on October 26 at 1430Z.

Resolves #439
Close #439

KateFriedman-NOAA committed:
Update GSI tag to gfsda.v16.1.5

- GSI update to turn on CrIS NPP radiances and include IASI Metop-C in operations

Refs: #439

KateFriedman-NOAA committed:
Add release notes for v16.1.5

Refs: #439

KateFriedman-NOAA committed:
Merge operations v16.1.4 into release/gfs.v16.1.5

Refs: #439

KateFriedman-NOAA committed:
Add mpirun flag to prepend rank

- added "-l" flag to mpirun command to prepend/insert the MPI process rank
at the beginning of all lines written to the standard output
- was tested in operations while investigating GFS ops failures
- include flag moving forward in operational system, starting with v16.1.5

Refs: #439

KateFriedman-NOAA committed:
Update release notes for v16.1.5 to include new CO2 fix files

- new 2022 CO2 fix files are bundled with DA update
- new fix files do not impact model results
- update release notes to mention CO2 fix files

Refs: #439

KateFriedman-NOAA committed:
Adjust text for CO2 files in release notes

- move text in release notes for new 2022 CO2 fix files
to more appropriate section

Refs: #439

KateFriedman-NOAA committed:
Final GFSv16.1.5 updates

- release note updates to mention new FV3 model tag that now includes
traceback compilation flags by default
- update checkout to use new FV3 tag: GFS.v16.0.17

Refs: #439

ADCollard committed:
Update use of historical files in parm/config/config.anal

Adding login in parm/config/config.anal reflecting the following
changes to observation use:
global_satinfo.txt.2020022012: Turn off MetOp-A MHS
global_satinfo.txt.2021052118: Turn off S-NPP CrIS
global_satinfo.txt.2021092206: Turn off Metop-A IASI
global_satinfo.txt.2021102612: Turn on MetOp-C IASI

Addresses issue #439

KateFriedman-NOAA committed:
Merge pull request #480 from ADCollard/release/gfs.v16.1.5

Update use of historical files in parm/config/config.anal

KateFriedman-NOAA commented:
Testing release branch on WCOSS-Dell to confirm it reproduces operations post-implementation.

HaixiaLiu-NOAA commented on docs/Release_Notes.gfs.v16.1.5.txt:
```diff
@@ -0,0 +1,117 @@
+ 
+PARM/CONFIG CHANGES
+
+* No change from GFS v16.1.4
```
Should we update the parm section since the file parm/config/config.anal was changed. The modification would not affect the operation since it only applies to the RUN_ENVIR == "emc" if blocks.

KateFriedman-NOAA replied:
Yes, good catch, perhaps something like this?

```
PARM/CONFIG CHANGES

* config.anal
  * update global_convinfo.txt notes and global_satinfo.txt if-blocks for retrospective dates
```

Please feel free to adjust that to what you'd prefer. I will commit the change, thanks!

KateFriedman-NOAA committed:
Update v16.1.5 release notes for config.anal update

- update the release notes to mention the change made to config.anal
- config.anal was updated to support correct global info files for
retrospective experiments

Refs: #439,#489

KateFriedman-NOAA committed:
Merge branch 'release/gfs.v16.1.5' of https://github.com/NOAA-EMC/global-workflow into release/gfs.v16.1.5

* 'release/gfs.v16.1.5' of https://github.com/NOAA-EMC/global-workflow:
  Update use of historical files in parm/config/config.anal
